server/service/src/item.rs

```diff
@@ -3,7 +3,7 @@ use repository::{Item, ItemFilter, ItemRepository, ItemSort, StorageConnectionMa
 
 use super::{get_default_pagination, i64_to_u32, ListError, ListResult};
 
-pub const MAX_LIMIT: u32 = 1000;
+pub const MAX_LIMIT: u32 = 5000;
```

Comment:
(Off-track) Was changed in FE for item query but not in BE so was giving LimitAboveMax error

Reply:
🙏  thanks! I changed f/e back to 1000, but this is much better!